packages/core/src/_test/ens/app/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { ponder } from "@/generated";
+import { ponder } from "../generated/index.js";
```

Comment:
Curious why this was necessary. Does the alias still work in the example projects?

Reply:
Actually not necessary after I added the vitest resolve. So reverting

Reply:
done